### PR TITLE
Package renaming

### PR DESCRIPTION
**Pull request by anwar6953**
Package renaming to work with CDAP 6.0.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>co.cask.re</groupId>
+  <groupId>io.cdap.re</groupId>
   <artifactId>dre</artifactId>
   <version>1.2.0</version>
```
Comment (CuriousVini) [inline pom.xml:22]: This should change as we are making modifications.

**Review by CuriousVini — COMMENTED** [review]
Also please update license year.

File: pom.xml (hunk 1 above)
Comment (anwar6953) [reply pom.xml:22]: We haven't released/published dre 1.2.0 yet, so I think this is fine.

**Comment by anwar6953** [comment]
Pushed a commit to update the end license year.

File: pom.xml (hunk 1 above)
Comment (CuriousVini) [reply pom.xml:22]: yea I think for this specific case its ok to merge to non-snapshot version

**Review by CuriousVini — APPROVED** [review]

**Pull request merged**